scripts/enum/item.lua

```diff
@@ -1992,16 +2114,20 @@ xi.item =
     DARK_CLUSTER                        = 4111,
     POTION                              = 4112,
     POTION_P1                           = 4113,
+    POTION_P2                           = 4114,
+    POTION_P3                           = 4115,
     HI_POTION                           = 4116,
     HI_POTION_P1                        = 4117,
     HI_POTION_P2                        = 4118,
     HI_POTION_P3                        = 4119,
     ETHER                               = 4128,
     ETHER_P1                            = 4129,
+    ETHER_P2                            = 4130,
+    ETHER_P3                            = 4131,
     HI_ETHER                            = 4132,
-    HI_ETHER_P1                         = 4133,
-    HI_ETHER_P2                         = 4134,
-    HI_ETHER_P3                         = 4135,
+    HI_ETHER_I                          = 4133,
```

Comment:
Hard no.  This scheme exists for a reason, and we do not use roman numerals when the db is +N

Reply:
Fair enough - and I get it but these enums were pulled from quests, missions, loot tables, dialog etc that will be submitted in the following weeks. prepping the battlefield so to speak so there are not 5 quests that all inevitably have merge conflicts because they sit for a while.

Reply:
> Fair enough - and I get it but these enums were pulled from quests, missions, loot tables, dialog etc that will be submitted in the following weeks. prepping the battlefield so to speak so there are not 5 quests that all inevitably have merge conflicts because they sit for a while.

That type of merge conflict should be trivial to deal with. Even editing the same file doesn't guarantee a conflict though. Usually its when 2 commits edit the same line. There is a good chance that if each quest PR contained the enumerations that specific quest needed that they'd actually make it through without a conflict, or trigger only the "made you look" type of conflict where git just makes you open it and tell it "this is fine" and there was no fire, save and resolve and go on your way. 

Keep this in mind going forward, even if another reviewer lets it pass this time. 

----

Also:

While we don't like preemptive adding to this particular enum, in other cases a way to avoid giant nearly unreviewable diffs is to break it into multiple commits (we're able to look at a specific commit). Its still better to just do it in multiple PRs but some cases may justify just packing 4-5 commits onto top each other (see the many many past item sql prs that have been happening - if it were all in a single commit in a single pr I'd flat out reject).